tools/init_repo.sh

```diff
@@ -10,11 +10,11 @@ git checkout develop
 for DIR in manifests microservices; do
   for FILE in `find ${CURRENT_DIR}/../${DIR}/ -type f`; do
     perl -pi -e "s|__LOADBALANCER_IP_ADDRESS__|${LOADBALANCER_IP_ADDRESS}|g" $FILE;
-    perl -pi -e "s|__TB_GITHUB_ORG_NAME__|${TB_GITHUB_ORG_NAME}|g" $FILE;
```

Comment:
It seems `${TB_GITHUB_ORG_NAME}` is correct here 🙏

Reply:
sorry. I fixed it.
I had made an innocent mistake.:pray::pray: